src/models/supplemental_accessors.jl

```diff
@@ -95,10 +95,67 @@ Returns 1/(R + jX) where R is resistance and X is reactance.
 """
 get_series_admittance(b::ACTransmission) = 1 / (get_r(b) + get_x(b) * 1im)
 
+"""
+Calculate the series admittance of a [`PhaseShiftingTransformer`](@ref) accounting for the tap ratio.
+For a phase-shifting transformer, the series admittance is calculated as the inverse of the
+complex impedance modified by the tap ratio, following the same pattern as the susceptance calculation:
+Y = 1/(tap * (R + jX)).
+The phase angle α affects the admittance matrix construction but not the series impedance magnitude directly.
```

Comment:
"The phase angle α affects the admittance matrix construction but not the series admittance value directly."